src/main/webapp/components/api/cache-creation-controller.service.js

```diff
@@ -366,8 +423,11 @@ angular.module('managementConsole.api')
             var propKey = keys[i];
             var propValue = prop[keys[i]];
             if (utils.isNotNullOrUndefined(propValue)) {
-              if (utils.isObject(propValue) && allowedObjects.indexOf(propKey) > -1) {
-                op[propKey] = propValue;
+              if (utils.isObject(propValue)) {
+                // Only process allowed objects, these should be objects which are dmr attributes not children
+                if (allowedObjects.indexOf(propKey) > -1) {
+                  op[propKey] = propValue;
+                }
```

Comment:
@vblagoje  FYI: It's necessary to nest the second check here, otherwise the objects are simply added further down as part of the if statement. This was a problem that I introduced several commits ago which had gone unnoticed, but resulted in lots of crap being sent to the server unnecessarily.